chat-app-express/src/model/userModel.js

```diff
@@ -19,7 +16,7 @@ const userSchema = new Schema({
         minLength: [3, 'Name should be at least 3 charcheters']
     },
     phone: {
-        type: Number,
+        type: String,
```

Comment:
why phone number is string?

Reply:
Some node issues Number can't be parsed into DB